dev-embedded/minipro/minipro-0.6.recipe

```diff
@@ -0,0 +1,64 @@
+SUMMARY="An open source program for controlling the MiniPRO TL866xx programmers"
+DESCRIPTION="Minipro is an Open Source tool intended to become a complete cross-platform
+replacement for the proprietary utility from Autoelectric. Currently it
+supports more than 13000 of target devices - including AVRs, PICs as
+well as a huge number of other microcontrollers and various memory
+chips.
+"
+HOMEPAGE="https://gitlab.com/DavidGriffith/minipro"
+COPYRIGHT="2014-2024 David Griffith"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://gitlab.com/DavidGriffith/minipro/-/archive/$portVersion/minipro-$portVersion.tar.gz"
+CHECKSUM_SHA256="16b4220b5fc07dddc4d1d49cc181a2c6a735c833cc27f24ab73eac2572c9304a"
+PATCHES="minipro-$portVersion.patchset"
+
+ARCHITECTURES="all !x86_gcc2"
+
+PROVIDES="
+	minipro = $portVersion
+	cmd:minipro = $portVersion
+	"
+
+REQUIRES="
+	haiku
+	lib:libusb_1.0
+	"
+
+PROVIDES_devel="
+	minipro_devel = $portVersion
+	devel:libminipro = $portVersion
+"
+
+REQUIRES_devel=$REQUIRES
+
+BUILD_REQUIRES="
+	haiku_devel
+	devel:libusb_1.0
+	"
+
+BUILD_PREREQUIRES="
+	cmd:ar
+	cmd:cp
+	cmd:gcc
+	cmd:git
```

Comment:
```suggestion
```

Reply:
`git` is used by the Makefile to generate a version file so I believe it's better to include it in the recipe.
See https://gitlab.com/DavidGriffith/minipro/-/blob/master/Makefile?ref_type=heads#L33.

Reply:
We create a temporary git repository for the build, which does not include the original commit and tags from minipro. So, the generated version will be meaningless and confusing. It's better to not pretend it comes from Minipro's own git repository, and generate the version otherwise (which the Makefile seems to know how to do already).

Reply:
I see, that make sense. I removed it then.